src/ItemOverlay/ItemOverlay.js

```diff
@@ -163,6 +165,42 @@ ItemOverlay.ItemOverlaySupport = {
 		{from: 'autoHideEnding', to: '$.ending.autoHide'}
 	],
 
+	handlers: {
+		onSpotlightFocused: 'spotFocused',
+		onSpotlightBlur: 'spotBlur'
+	},
+
+	/**
+	* @private
+	*/
+	overlayShowing: function(showing) {
+		if (this.autoHideBeginning) {
+			this.$.beginning.set('showing', showing);
+		}
+
+		if (this.autoHideEnding) {
+			this.$.ending.set('showing', showing);
+		}
+	},
+
+	/**
+	* @private
+	*/
+	spotFocused: function() {
+		this.overlayShowing(true);
+		this.resize();
```

Comment:
We would only need to resize if `autoHideBeginning` or `autoHideEnding` were true, right?